smithy-model/src/main/java/software/amazon/smithy/model/loader/IdlModelParser.java

```diff
@@ -1126,7 +1126,7 @@ private void parseApplyStatement() {
         expect('p');
         expect('l');
         expect('y');
-        sp();
+        rsp();
```

Comment:
Too bad that the names of the methods have diverged from the name of the grammar productions, but that's not related to this change.